crates/torii/grpc/src/server/mod.rs

```diff
@@ -147,34 +147,31 @@ impl DojoWorld {
         let mut subs = Vec::with_capacity(queries.len());
         for query in queries {
             let clause: KeysClause = query
+                .clone()
```

Comment:
I think we can modify the input of this to be the KeysClause and simplify anything a lot